src/layouts/Resources.jsx

```diff
@@ -1,378 +1,166 @@
-            : null}
-
-          {/* main section starts here */}
-          <div className={contentStyles.mainSection}>
-            <div className={contentStyles.sectionHeader}>
-              <h1 className={contentStyles.sectionTitle}>Resources</h1>
-              { resourceRoomName
-                ? (
-                  <ResourceCategoryModal
+  return (
+    <>
+      <Header />
+      {/* main bottom section */}
+      <div className={elementStyles.wrapper}>
+        <Sidebar siteName={siteName} currPath={location.pathname} />
+        {/* main section starts here */}
+        <div className={contentStyles.mainSection}>
+          {/* Page title */}
+          <div className={contentStyles.sectionHeader}>
+            <h1 className={contentStyles.sectionTitle}>Resources</h1>
+          </div>
+          {
+            isLoading ? 'Loading Resources...'
+            :
+              resourceRoomName
+              ? <>
+                  {/* Category title */}
+                  <div className={contentStyles.segment}>
+                    Resource Categories
+                  </div>
+                  {/* Info segment */}
+                  <div className={contentStyles.segment}>
+                    <i className="bx bx-sm bx-info-circle text-dark" />
+                    <span><strong className="ml-1">Note:</strong> Categories cannot be empty, create a resource first to create a Category.</span>
+                  </div>
+                  {/* Categories */}
+                  <div className={contentStyles.folderContainerBoxes}>
+                    <div className={contentStyles.boxesContainer}>
+                      {
+                        resourceFolderNames 
+                        ? 
+                          resourceFolderNames.length > 0
+                          ? resourceFolderNames.map((resourceCategory, collectionIdx) => (
+                              <FolderCard
+                                displayText={prettifyResourceCategory(resourceCategory)}
+                                settingsToggle={() => {}}
+                                key={resourceCategory}
+                                isHomepage={false}
+                                isCollection={false}
+                                siteName={siteName}
+                                category={resourceCategory}
+                                itemIndex={collectionIdx}
+                              />
+                            ))
+                          : 'No Resource Categories. Create a resource to add a Category.'
+                        : 'Loading Resource Categories...'
+                      }
+                    </div>
+                  </div>
+                  {/* Segment divider  */}
+                  <div className={contentStyles.segmentDividerContainer}>
+                    <hr className="invisible w-100 mt-3 mb-5" />
+                  </div>
+                  {/* Pages */}
+                  <CollectionPagesSection
```

Comment:
There is a missing `isResource` prop that isn't being passed into the `CollectionPagesSection` component currently from the `Resources` layout. This results in a a situation where collections, instead of resource categories, are retrieved when trying to create a new resource

Steps to reproduce:
1. Delete all resource pages and resource room
2. Create a new resource room (i'm using the `a-test` repository)
3. Try to create a new resource. In the category dropdown, there should be no options, but the `ComponentSettingsModal` displays `test`, which is in fact a collection and not a resource category

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/207/commits/5f587ba310528d7d11a8acedf63b5e1620d4a241